front/src/pages/AddItem/AddItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './AddItem.module.scss';
+
+const AddItem = () => {
+  const handleFormSubmit = (e: React.SyntheticEvent) => {
+    e.preventDefault();
+    const target = e.target as HTMLFormElement;
+    const objectOfFields = Object.fromEntries(new FormData(target));
```

Comment:
should there be api call?

Reply:
@skyboyer 
Yes, I have a task for it in my notebook. I should have left a todo-comment here to implement this logic.